dependencies/install_onnx.sh

```diff
@@ -0,0 +1,21 @@
+if [ ! -f /usr/local/lib/libonnxruntime.so ]; then
+
+  echo "Downloading onnxruntime (this might take a while) ..."
+  git clone -q --recursive https://github.com/Microsoft/onnxruntime
+  cd onnxruntime
+
+  # Checkout v1.6.0 (https://github.com/microsoft/onnxruntime/releases/tag/v1.6.0)
+  git checkout 718ca7f > /dev/null 2>&1
```

Comment:
Just a side note, this should be the same as doing directly 
```suggestion
  git checkout v1.6.0 > /dev/null 2>&1
```